packages/component-library/src/components/plmg-tooltip/plmg-tooltip.tsx

```diff
@@ -0,0 +1,307 @@
+import { Component, h, Prop, State, Watch } from '@stencil/core';
+
+import {
+  isArrowPosition,
+  isArrowSide,
+  isBackgroundColor,
+  PlmgTooltipArrowPositions,
+  PlmgTooltipArrowSides,
+  PlmgTooltipBackgroundColors,
+} from './plmg-tooltip.types';
+
+@Component({
+  tag: 'plmg-tooltip',
+  styleUrl: 'plmg-tooltip.scss',
+  shadow: true,
+})
+export class Tooltip {
+  private abortTooltipListener: AbortController;
+  private targetHTMLElement: HTMLElement;
+
+  /**
+   * Store tooltip visibility.
+   *
+   * Default hidden
+   */
+
```

Comment:
I don't like all these empty lines between the JSDoc and the attribute. :innocent: 
Can you delete them? Or is your IDE applying this format automatically?